xrfeitoria/cmd/blender/vis_smplx.py

```diff
@@ -0,0 +1,193 @@
+"""Install a blender plugin with a command line interface.
+
+>>> xf-install-plugin --help
+>>> xf-render {} [-o {output_path}]
+"""
+
+from pathlib import Path
+from textwrap import dedent
+from typing import Tuple
+
+import xrfeitoria as xf
+from xrfeitoria.rpc import remote_blender
+from xrfeitoria.utils import setup_logger
+
+try:
+    import numpy as np
+    from typer import Argument, Option, Typer
+    from typing_extensions import Annotated
+
+    from xrfeitoria.utils.anim.motion import Motion, SMPLMotion, SMPLXMotion
+except (ImportError, NameError):
+    pass
+
+app = Typer(pretty_exceptions_show_locals=False)
+
+
+@remote_blender()
+def add_smplx(betas: 'Tuple[float, ...]' = [0.0] * 10, gender: str = 'neutral') -> str:
```

Comment:
Use mutable types (list) as default values could cause bugs.
```suggestion
def add_smplx(betas: 'Tuple[float, ...]' = tuple([0.0] * 10), gender: str = 'neutral') -> str:
```

Reply:
Fixed in [`58f1ada` (#15)](https://github.com/openxrlab/xrfeitoria/pull/15/commits/58f1ada79c17df94c10a55633a41dc965a4a2816#diff-c4f7228b2fbf2d2bf2ef2406413575841160c9320874219b14282abe778e605dR28).
Modify it to `betas: 'Tuple[float, ...]' = (0.0,) * 10`